feature/settings/src/main/java/com/google/jetpackcamera/settings/ui/SettingsComponents.kt

```diff
@@ -622,6 +623,33 @@ fun StabilizationSetting(
     )
 }
 
+@Composable
+fun MuteRecordingSetting(
+    modifier: Modifier = Modifier,
+    mutedUiState: MuteAudioUiState,
+    setDefaultMuted: (Boolean) -> Unit
+) {
+    SwitchSettingUI(
+        modifier = modifier,
+        title = stringResource(id = R.string.mute_audio_title),
+        description = if (mutedUiState is MuteAudioUiState.Enabled) {
+            if (mutedUiState.isMuted) {
+                stringResource(R.string.mute_selector_on)
+            } else {
+                stringResource(R.string.mute_selector_off)
+            }
+        } else {
+            TODO("mute toggle currently has no disabled criteria")
```

Comment:
What about if the audio permission wasn't granted?

Reply:
<img src="https://media.giphy.com/media/vgUl8kBguVg2UbmI2U/giphy.gif?cid=790b76111wpaw8r6cjzlbyfj2grl18klm2i6ijrl1ro89yt0&ep=v1_stickers_search&rid=giphy.gif&ct=s" height="100" />

For now I'll disable if the permission isnt granted.
But I guess it could display some UI to prompt the user to grant the permission. That can be addressed a later PR though, since there would also need a case to navigate to the app settings page.